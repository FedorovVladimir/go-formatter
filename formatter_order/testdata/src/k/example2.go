package main

func f() { // want "formatter_order"
}

// s - s // want "formatter_order"
type s struct {
}

// s2 - s2 // want "formatter_order"
type (
	s2 struct {
	}
)
