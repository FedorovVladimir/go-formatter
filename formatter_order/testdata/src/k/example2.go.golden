package main

// s - s // want "incorrect declaration order"
type s struct {
}

// s2 - s2 // want "incorrect declaration order"
type (
	s2 struct {
	}
)

func f() { // want "incorrect declaration order"
}
